cognitoclient/cognito.go

```diff
@@ -0,0 +1,11 @@
+package cognitoclient
```

Comment:
As mentioned consider renaming this package to avoid the stutter. I think `cognito` would be appropriate but whatever works is fine.